doc/api/http.md

```diff
@@ -920,6 +920,13 @@ Stops the server from accepting new connections. See [`net.Server.close()`][].
 Starts the HTTP server listening for connections.
 This method is identical to [`server.listen()`][] from [`net.Server`][].
 
+Possible signatures:
```

Comment:
I don't know if it's makes sense as it is already linked to https://nodejs.org/dist/latest/docs/api/net.html#net_server_listen which list the possible signatures. However the last one can be improved. Should we change this PR and tackle that instead?

Reply:
I see your point, maybe we could move the spec discussion to this issue https://github.com/nodejs/node/issues/16300.

Was you suggestion to add the last signature to net.Server.listen() instead?

Reply:
Yes, exactly. I proposing this basically because of https://github.com/nodejs/node/commit/278f65351231f9fa034220780fba8aae940c1b9a.

Reply:
Is that very different from the last signature in net.Server.listen()?

Perhaps the initial issue does not need to be addressed.

Reply:
Yes, it's quite different because it makes it clear which parameters will be used in each scenario. The signature in net.Server.listen makes it seem like it's possible to leave out port & host and include backlog, but it's not actually. The signature @joyeecheung came up with is more precise.